C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackBotWorker.cs

```diff
@@ -104,8 +199,36 @@ public async Task<object> ReplyPrivate(object source, object resp)
         /// </summary>
         /// <param name="source">An incoming message object of type `interactive_message`</param>
         /// <param name="resp">A new or modified message that will replace the original one</param>
-        public async Task<object> ReplyInteractive(object source, object resp)
+        public async Task<object> ReplyInteractive(IBotkitMessage source, IBotkitMessage resp)
         {
+            if(!((source.IncomingMessage.ChannelData as dynamic).ResponseUrl))
```

Comment:
The original code `if (!src.incoming_message.channelData.response_url)` is checking if `response_url` is (not) not null.

In C# cannot check if a property exists or not like this. This should be changed to:
``` C#
if(((source.IncomingMessage.ChannelData as dynamic).ResponseUrl) != null)
{
    throw new Exception("No response_url found in incoming message");
}
```

Reply:
Done. Thanks!